src/components/DetailedComparisonItem.jsx

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable no-restricted-globals */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCaretRight } from '@fortawesome/free-solid-svg-icons';
+
+import {
+  FlexDiv,
+  WrapperCaretIconGrey,
+  WrapperTitleAccordion,
+  WrapperHeadAccordion,
+  DetailsIndicators,
+  DetailItem,
+  FlexDivAccordionDetailed,
+  DetailedDiv,
+  DetailedNoteCountry,
+  NoteDivColor,
+} from '../styles/containers';
+import {
+  TitleIndicatorDetail,
+  NoteColor,
+  DetailIndicatorTitle,
+} from '../styles/texts';
+
+const themesWithNotes = [
+  { theme: 'Ecologie', note: '56' },
+  { theme: 'Droits fondamentaux', note: 'XX' },
+  { theme: 'Société', note: 'XX' },
+];
+
+const items = [
+  { title: 'Premier indicateur', notes: ['10', 'XX', 'XX'] },
+  { title: 'Second indicateur', notes: ['70', 'XX', 'XX'] },
+  { title: 'Troisième indicateur', notes: ['10', 'XX', 'XX'] },
+  { title: 'Quatrième indicateur', notes: ['40', 'XX', 'XX'] },
+];
+
+const DetailedComparisonItem = ({
+  nbCountries,
+  handleClick,
+  titleTheme,
+  id,
+}) => {
+  const width = `calc(100%/${nbCountries})`;
+
+  return (
+    <>
+      <FlexDiv>
+        <FlexDivAccordionDetailed column all>
+          <WrapperHeadAccordion
+            padDetailed
+            topOnly
+            pointer
+            onClick={() => handleClick(id)}
+          >
+            <WrapperTitleAccordion>
+              <WrapperCaretIconGrey id={`toRotate${id}`}>
+                <FontAwesomeIcon icon={faCaretRight} />
+              </WrapperCaretIconGrey>
+              <TitleIndicatorDetail>{titleTheme}</TitleIndicatorDetail>
+            </WrapperTitleAccordion>
+          </WrapperHeadAccordion>
+          <FlexDiv>
+            {themesWithNotes.map((themeNote) => {
+              return (
+                <DetailedNoteCountry style={{ width }} borderRight>
```

Comment:
there is a width without value in the inline style

Reply:
This width refers to the width defined at the beginning of the component : " const width = `calc(100%/${nbCountries})`; "